recipes/spectrumdevice/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set owner = "KCL-BMEIS" %}
+{% set name = "spectrumdevice" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ owner }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5e044fbf034fbcf6ba953d4d156e6479de4624baa72c7f0ea21abc133942288f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - numpy >=1.21.4
+    - python >=3.8
+
+test:
+  imports:
+    - spectrum_gmbh
+    - spectrumdevice
+    - tests
+    - example_scripts
```

Comment:
This passing means there are "tests" and "example_scripts" directories being installed globally in the environment. This is prone to collision with other packages (not to mention messy). It would be good to solve this, either upstream or here with a patch to `setup.py` to explicitly list the "spectrum_gmbh" and "spectrumdevice" as the packages to install.

Reply:
Good catch, thanks. I was using `setup.cfg` to exclude packages (see [here](https://github.com/KCL-BMEIS/spectrumdevice/blob/main/setup.cfg)).

I changed `setup.py` to explicitly exclude the two problem packages (see [here](https://github.com/KCL-BMEIS/spectrumdevice/blob/main/setup.py)):
```
packages=setuptools.find_packages(".", exclude=["tests", "example_scripts"]))
```
and re-ran the conda build locally in docker (using `run_docker_build.sh`), keeping the `import tests` and `import example_scripts` tests in `meta.yaml`, hoping it would fail. However, the build completed successfully, so apparently the packages were still installed. Any idea why?

Reply:
The short answer is that getting exclude to work properly is tricky and sometimes impossible. I'd go with an explicit `include` instead.

Reply:
I couldn't make exclude or include work. Instead, I have changed the source to be the PyPI package (which doesn't include tests or examples), and configured the conda build to download the tests separately from GitHub.

This is now ready for review again, thanks for your help.